helm/templates/agent-daemonset.yaml

```diff
@@ -71,65 +71,46 @@ spec:
             {{- if .Values.server.env }}
             {{- toYaml .Values.server.env | indent 12}}
             {{- end }}
-          lifecycle:
-            postStart:
-              exec:
-                command:
-                  - /checks/cloudzero-agent-validator
-                  - diagnose
-                  - post-start
-                  - -f
-                  - /checks/app/config/validator.yml
-            preStop:
-              exec:
-                command:
-                  - /checks/cloudzero-agent-validator
-                  - diagnose
-                  - pre-stop
-                  - -f
-                  - /checks/app/config/validator.yml
           args:
-            {{ toYaml .Values.server.args | nindent 12}}
-            {{- if .Values.server.agentMode }}
+            - --config.file=/etc/config/prometheus.yml
+            - --web.enable-lifecycle
+            - --web.console.libraries=/etc/prometheus/console_libraries
+            - --web.console.templates=/etc/prometheus/consoles
             - --enable-feature=agent
-            {{- end }}
           ports:
             - containerPort: 9090
           readinessProbe:
             httpGet:
               path: /-/ready
               port: 9090
               scheme: HTTP
-            initialDelaySeconds: {{ default 30 .Values.server.readinessProbeInitialDelay }}
-            periodSeconds: {{ default 5 .Values.server.readinessProbePeriodSeconds }}
-            timeoutSeconds: {{ default 4 .Values.server.readinessProbeTimeout }}
-            failureThreshold: {{ default 3 .Values.server.readinessProbeFailureThreshold }}
-            successThreshold: {{ default 1 .Values.server.readinessProbeSuccessThreshold }}
+            initialDelaySeconds: 30
+            periodSeconds: 5
+            timeoutSeconds: 4
+            failureThreshold: 3
+            successThreshold: 1
```

Comment:
I'm not super opposed to removing these, but I think it's worth noting that being able to configure these things can be helpful for debugging. but, I've never had to do that for this chart specifically, so 🤷